charts/nginx/templates/nginx-config-role.yaml

```diff
@@ -44,4 +44,26 @@ rules:
       - get
       - create
       - update
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    resourceNames:
+      - ingress-controller-leader-{{ template "nginx.fullname" . }}
+    verbs:
+      - get
+      - update
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - create
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
```

Comment:
Doesn't this make 1.19 of K8s unsupported. That means we are no longer supporting 1.19 on our 0.29.x release?

Reply:
removing support only for 1.19.0